### DISCUSSION TITLE
Unable to style day cell

### DISCUSSION THREAD

**Discussion by t-zander** [gpbl/react-day-picker#1146] [Support] [answered]
**Describe the bug**
I am unable to style date cell, in any way. I've used `modifiersStyles`. 
I need it because for some reason I have white color of days and white background on hover.
As far I got it, I can use any of [these](https://react-day-picker.js.org/api/DayPicker#modifiersStyles)

Here is the code:
```
const styles = {
    today: {
      color: theme.palette.primary.dark // it comes from MUI react (I haven't found any clear way to combine their makeStyles utility with react-day-picker
    },
    selected: {
      backgroundColor: theme.palette.primary.main
    },
    day: {
      color: "red" // doesn't work
    }
  };

<DayPicker
        selectedDays={selectedDays}
        onDayMouseEnter={handleDayMouseEnter}
        onDayClick={handleDayClick}
        numberOfMonths={2}
        months={MONTHS}
        weekdaysLong={WEEKDAYS_LONG}
        weekdaysShort={WEEKDAYS_SHORT}
        firstDayOfWeek={1}
        modifiersStyles={styles}
      />
```

**To Reproduce**
https://codesandbox.io/s/react-day-picker-base-8b440

**Expected behavior**
**Screenshots**
https://prnt.sc/q4o2bq
**Additional context**
I am on Next.js
I am using material-ui react
No css modules, only MUI tool called makeStyles()

**Comment by KyorCode** [comment, marked as answer]
I don't think 'day' is a default modifier, therefor not being able to be styled this way.

However you can use renderDay to use your own component, which you can style anyway you want.
https://react-day-picker.js.org/api/DayPicker#renderDay
